docs/src/modules/ROOT/pages/introduction/introduction.adoc

```diff
@@ -112,63 +113,77 @@ As you can see in the examples, most instances have a lot more possible solution
 Timefold supports several optimization algorithms to efficiently wade through that incredibly large number of possible solutions.
 Depending on the use case, some optimization algorithms perform better than others, but it's impossible to tell in advance. **With Timefold, it is easy to switch the optimization algorithm**, by changing the solver configuration in a few lines of XML or code.
 
-[[requirements]]
-== Requirements
+[[timefoldSolverStatus]]
+== Status of Timefold Solver
+
+Timefold Solver is 100% pure Java^TM^ and runs on Java {java-version} or higher.
+It xref:integration/integration.adoc#integration[integrates very easily] with other Java^TM^ technologies.
+Timefold Solver works on any Java Virtual Machine and is compatible with the major JVM languages and all major platforms.
+
+image::introduction/compatibility.png[align="center"]
+
+Timefold Solver is stable, reliable and scalable.
+It has been heavily tested with unit, integration, and stress tests, and is used in production throughout the world.
+One example handles over 50 000 variables with 5000 values each, multiple constraint types and billions of possible constraint matches.
+
+We offer two editions of Timefold Solver.
+
+[[communityEdition]]
+=== Timefold Solver Community Edition
 
 Timefold Solver Community Edition is _open source_ software,
 released under http://www.apache.org/licenses/LICENSE-2.0.html[the Apache License 2.0].
 This license is very liberal and allows reuse for commercial purposes.
 Read http://www.apache.org/foundation/licence-FAQ.html#WhatDoesItMEAN[the layman's explanation].
 
-Timefold is 100% pure Java^TM^ and runs on Java {java-version} or higher.
-It xref:integration/integration.adoc#integration[integrates very easily] with other Java^TM^ technologies.
-Timefold is available in <<useWithMavenGradleEtc,the Maven Central Repository>>.
-
-Timefold works on any Java Virtual Machine and is compatible with the major JVM languages and all major platforms.
+Timefold Solver Community Edition is available in <<useWithMavenGradleEtc,the Maven Central Repository>>.
+It is and will always be free.
+The overwhelming majority of solver features will always be available in the Community Edition.
```

Comment:
Should we clearly say that every feature apart from scalability features is included?

Reply:
The enterprise section has a link to the pricing page, which lays it out clearly. (Or rather it will.) I'd rather avoid repetition.